components/molecules/ContributorProfileInfo/contributor-profile-info.tsx

```diff
@@ -69,16 +69,25 @@ const ContributorProfileInfo = ({
                 "I am an open source developer with a passion for music and video games. I strive to improve the open source community and am always looking for new ways to contribute."}
             </p>
             <div className="flex flex-col text-sm mt-2 text-light-slate-9 gap-2">
-              {displayLocalTime && <span className="flex gap-2 items-center">
-                <FiClock className="text-light-slate-9" /> Local time: 12:32 PM
-              </span>}
+              {displayLocalTime && (
+                <span className="flex gap-2 items-center">
+                  <FiClock className="text-light-slate-9" /> Local time: 12:32 PM
```

Comment:
```suggestion
                  <FiClock className="text-light-slate-9" /> Local time: 12:32 PM
```
weird that this is still hardcoded 😅 - are you able to fix or hide this?

Reply:
Pushed a fix